lib/build_tindex_master.py

```diff
@@ -260,7 +260,8 @@ def main():
                     ends_with_str = ".parquet"
             if "AGB" in TYPE or 'HT' in TYPE:
                 if user is None: user = 'lduncanson'
-                dps_out_searchkey_list = [f"{user}/dps_output/{alg_name}/{args.dps_identifier}/{dps_year}/{dps_month}/{format(d, '02')}/**/*.tif" for d in range(args.dps_day_min, args.dps_day_max + 1) for dps_month in dps_month_list for dps_year in dps_year_list]
+                dps_out_searchkey_list = [f"{user}/dps_output/{alg_name}/{args.dps_identifier}/{dps_year}/{dps_month}/{format(d, '02')}/**/*_2020_*.tif" for d in range(args.dps_day_min, args.dps_day_max + 1) for dps_month in dps_month_list for dps_year in dps_year_list]
```

Comment:
As you mentioned, 2020 is not needed.
I'd say let's update the search regex for the new naming convention, so that we only look for `*.tif` with 7 digit `tile_num` in the filename. That looks something like `*_[0-9]{7}.tif` but I don't remeber the syntax.

Reply:
Actually, `s3fs.glob`  doesn't support the range syntax `{7}`. So for now, we can just do `*_???????.tif` (matches 7 chars in addition to digits) or  `*_[0-9][0-9][0-9][0-9][0-9][0-9][0-9].tif` to be more precise.